projects/swimlane/ngx-ui/src/lib/components/json-editor/json-editor.component.ts

```diff
@@ -27,17 +28,27 @@ export class JsonEditorComponent implements OnChanges {
   @Input()
   label: string;
 
+  @Input()
+  typeCheckOverrides: any;
```

Comment:
Is this always mandatory? Looks like it could perhaps be optional?

Reply:
Yes, made it optional. Is there anything in the angular template compiler that checks whether required params are passed? I haven't encountered that anywhere yet.

Reply:
Yes, if you omit an Input on the template that is not optional the compiler should give an error. At least the aot compiler should.